UPGRADE-4.0.md

```diff
@@ -228,6 +228,8 @@ Form
    ));
    ```
 
+ * Removed `ChoiceLoaderInterface` implementation in `TimezoneType`.
```

Comment:
btw, while at it, is it worth to do this deprecation for other types? LanguageType and co. could do the same approach =/

It seems a way simpler implem. (CallbackChoiceLoader) compared to to ArrayChoiceLoader.